components/derive/src/exonum_interface.rs

```diff
@@ -139,22 +140,24 @@ impl ServiceMethodDescriptor {
 
 #[derive(Debug, FromMeta)]
 #[darling(default)]
-struct ExonumServiceAttrs {
+struct ExonumInterfaceAttrs {
     #[darling(rename = "crate")]
     cr: RustRuntimeCratePath,
+    id_auto_increment: bool,
```

Comment:
Nit / bikeshedding: Is the name intentionally long? 🙃 I'd prefer something shorter, like `auto_ids`.

Reply:
Yeah, the idea was to make it less pretty. However, I don't mind `auto_ids`.